targets/XPlatCppSdk/source/cppsdk/include/playfab/PlayFabHttp.h

```diff
@@ -41,12 +47,16 @@ namespace PlayFab
         static void MakeInstance();
         ~PlayFabHttp() override;
 
-        void AddRequest(const std::string& urlPath, const std::string& authKey, const std::string& authValue, const Json::Value& requestBody, RequestCompleteCallback internalCallback, SharedVoidPointer successCallback, ErrorCallback errorCallback, void* customData) override;
+        virtual void AddPostRequest(const std::string& url, std::unordered_map<std::string, std::string> headers, const std::string& requestBody, std::function<void(CallRequestContainerBase&)> callback, ErrorCallback errorCallback = nullptr, void* extrabuffer = nullptr) override;
+
         size_t Update() override;
+
     private:
         PlayFabHttp(); // Private constructor, to enforce singleton instance
         PlayFabHttp(const PlayFabHttp& other); // Private copy-constructor, to enforce singleton instance
 
+        virtual void InternalAddRequest(const std::string& urlPath, std::unordered_map<std::string, std::string> headers, /*const std::string& authKey, const std::string& authValue,*/ const Json::Value& requestBody, std::function<void(CallRequestContainerBase&)> internalCallback, ErrorCallback errorCallback, void* customData) override;
```

Comment:
>unordered_map [](start = 73, length = 13)

Are we passing a copy of maps with every call or a reference or a pointer would be enough?